tool/state-processor/src/processor.rs

```diff
@@ -393,7 +393,9 @@ impl<R> State<R> {
 	pub fn reserve(&mut self, who: &str, amount: u128) {
 		self.mutate_account(who, |a| {
 			if a.data.free < amount {
-				log::warn!("`Account({who})` can't afford the latest runtime reservation amount");
+				// No longer needed, since we release all reservations.
+				// log::warn!("`Account({who})` can't afford the latest runtime reservation
+				// amount");
```

Comment:
Although you added some comment, I did not understand the reason. I know we release all reservations, but this `reserve` function is used to re-reserve to the new chain state. Some accounts register the identity for a long time, but the current balance cannot afford the latest deposit amount.

Reply:
#994